samples/yesno/README.md

```diff
@@ -0,0 +1,183 @@
+
+
+**What does this sample demonstrate:**
+- Run [kaldi asr toolkit yesno sample](https://github.com/kaldi-asr/kaldi/tree/master/egs/yesno) in Azure Machine Learning pipeline.
+- Create tests by mocking Azure ML SDK.
+
+**What doesn't this sample demonstrate**:
+- ML model deployment.
+
+## Run kaldi sample in Azure ML pipeline
+
+### Additional software for Azure ML pipeline
+To run Kaldi sample, you need to have [Kaldi ASR Toolkit](https://github.com/kaldi-asr/kaldi) and its dependencies in compute cluster. Azure ML provides out of box environments as base image when you run Azure ML pipeline, but non of them include Kaldi ASR Toolkit.
+
+Whenever you need additional software, you can create custom docker image and use it as custom base image. This sample demonstrate how you can provision and use custom image in Azure ML pipeline.
+
+### Python wrapper for Azure ML pipeline
+The [yesno sample](https://github.com/kaldi-asr/kaldi/tree/master/egs/yesno) uses bash script to train the model. However, Azure ML pipeline requires python code to execute steps. This sample demonstrate how you can write python wrapper code to run underline shell script as part of Azure ML pipeline step.
+
+# Getting Started
+
+- [Prerequisites](#Prerequisites)
+- [Running locally](#Running-locally)
+## Prerequisites
+
+- [Create Azure Resources](#Create-Azure-Resources)
+- [Build and push custom base image](#Build-and-push-custom-base-image)
+- [Prepare input data](#Prepare-input-data)
+- [Add Azure ML compute, datastore and datasets](#Add-Azure-ML-compute-datastore-and-datasets)
+
+### Create Azure Resources
+
+1. Whether you run this project locally or in Azure DevOps CI/CD pipelines, the code needs to get Azure ML context for remote or offline runs. Create Azure resources as documented [here](../../common/infrastructure/README.md).
+
+1. Review the folder structure explained [here](../../README.md#repo-structure).
+
+### Build and push custom base image
+
+This sample requires Kaldi ASR Toolkit which is not avaiable for out of box Azure Machine Learning Environments. You can create custom docker images which contains all requires dependencies by yourself nad use it as Azure Machine Learning Environment. See [Create & use software environments in Azure Machine Learning](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-use-environments) for more detail.
+
+This sample contains [Dockerfile](./azureml_environment/Dockerfile) to build a custom base image which contains Kaldi ASR Toolkit and dependencies. Follow the steps below to build the image and push it to container registory. We recommend you to run separate VM which contains docker engine to build the image as it may take 30-60 minutes depending on specs.
+
+1. Download SRILM from http://www.speech.sri.com/projects/srilm/download.html into same folder as Dockerfile.
```

Comment:
I suggest to refer also to the directory where the Dockerfile is located. Or just state `Download ... into folder /samples/yesno/azureml_environment`

Reply:
Also it looks like the downloaded file needs to be renamed:
![image](https://user-images.githubusercontent.com/29010283/108144719-8bdd9400-710d-11eb-88d0-c59aa43b7253.png)

Reply:
okay. please review it again.